src/include/ipc4/notification.h

```diff
@@ -45,7 +45,7 @@ enum sof_ipc4_notification_type {
 #define SOF_IPC4_GLB_NOTIFY_DIR_MASK		BIT(29)
 #define SOF_IPC4_REPLY_STATUS_MASK		0xFFFFFF
 #define SOF_IPC4_GLB_NOTIFY_TYPE_SHIFT		16
-#define SOF_IPC4_GLB_NOTIFY_MSG_TYPE_SHIFT		24
+#define SOF_IPC4_GLB_NOTIFY_MSG_TYPE_SHIFT	24
 
```

Comment:
This is not related to this patch? Could be easily added as a separate patch even a separat PR.